helix-term/src/keymap.rs

```diff
@@ -648,6 +648,7 @@ impl Default for Keymaps {
 
             "space" => { "Space"
                 "f" => file_picker,
+                "F" => file_picker_in_directory_of_active_buffer,
```

Comment:
I wonder what's the key for this in doom emacs.

Reply:
Was it a suggestion or you just mentioned it? I would not sacrifice the current `space f` shortcut as a prefix, it is too common.

Reply:
#361

Reply:
Should I maybe modify the PR to contains only the command implementation without the default key bindings, so the feature can be used with custom configuration while the conversation about the keybindings is going?

Reply:
We can easily fill up the keybindings and later we will have to do like what doom emacs did, more like current `space w` which requires only an extra key, I think doing this earlier allows us to reduce breaking changes later.

Reply:
I agree that `space f` should be kept as is. I don't think we need `space f F`, `space F` is enough.

Reply:
> I don't think we need space f F, space F is enough.

Yes, for now it's fine. But later when we have as much `space f *` like in doom emacs we might have to reconsider this.

Reply:
Yeah I'm not sure whether we will add all of those, helix isn't doom emacs. Emacs often relies on longer key sequences (`SPC h d h` etc), but we don't

Reply:
I can't actually find any documentation on `SPC f` https://github.com/hlissner/doom-emacs/search?p=1&q=spc+f

Reply:
Just to throw some new info out there I've raised a PR related to a similar picker command `file_picker_at_cwd` which opens the file picker at the current working directory: #1600

If that PR ends up being merged there would be 3 different file picker related commands including the `file_picker_in_directory_of_active_buffer` being discussed in this PR.

Does this new info impact your thoughts about the keymapping of commands and whether or not it would be appropriate to add a submenu to group the "picker" related commands?